python-checks/src/test/java/org/sonar/python/checks/ExceptionNotThrownCheckTest.java

```diff
@@ -31,5 +32,20 @@ public void test() {
       new ExceptionNotThrownCheck());
   }
 
+  @Test
+  public void quick_fix_test() {
+    var input = "def func():\n" +
+      "  NotImplementedError()\n";
+    var expected = "def func():\n" +
+      "  raise NotImplementedError()\n";
+    PythonQuickFixVerifier.verify(new ExceptionNotThrownCheck(), input, expected);
+  }
+
+  @Test
+  public void quick_fix_message_test() {
+    var input = "def func():\n" +
+                "  NotImplementedError()\n";
+    PythonQuickFixVerifier.verifyQuickFixMessages(new ExceptionNotThrownCheck(), input, ExceptionNotThrownCheck.QUICK_FIX_MESSAGE);
```

Comment:
Same as in other PR, I think this assertion is simple enough that you can just put it in the previous test and just keep a single test.